charts/deploy/charts/grafana-operator/templates/Grafana.yaml

```diff
@@ -86,10 +86,8 @@ spec:
               properties:
                 ports:
                   type: array
-                  description: Override default ports
```

Comment:
did you remove these descriptions intentionally?

Reply:
![argocd-sync-grafana](https://user-images.githubusercontent.com/712608/87612991-60e22a80-c74f-11ea-8b14-a122e6decaa5.png)

yeah .. this is annoying - argocd sync fails with these descriptions there (1.6.1 argocd) .. CRD's are fiddly i know as far as sync is concerned will see if there is workaround on argocd side (ignoredifferences etc) ...

Reply:
reverted these .. looking for argocd side change instead